eks/cluster-loader/clusterloader2/cluster-loader.gotmpl

```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: clusterloader2
+  name: cl2
```

Comment:
I prefer the more verbose clusterloader2 namespace. If we establish a pattern where addon name and namespace of the resources have parity, that will make things easier to work with.

Reply:
I originally made it cl2 for easier debugging purposes, but I like the idea of standardizing the namespaces.